tests/b026.py

```diff
@@ -0,0 +1,16 @@
+"""
+Should emit:
+B026 - on lines 14, 15, 16
+"""
+
+
+def foo(bar, baz, bam):
+    pass
+
+
+foo("bar", "baz", bam="bam")
+foo("bar", baz="baz", bam="bam")
+foo(bar="bar", baz="baz", bam="bam")
+foo(bam="bam", *["bar", "baz"])
+foo(baz="baz", bam="bam", *["bar"])
+foo(bar="bar", baz="baz", bam="bam", *[])
```

Comment:
```suggestion
bar_bar = ["bar", "baz"]
foo("bar", "baz", bam="bam")
foo("bar", baz="baz", bam="bam")
foo(bar="bar", baz="baz", bam="bam")
foo(bam="bam", *["bar", "baz"])
foo(bam="bam", *bar_baz)
foo(baz="baz", bam="bam", *["bar"])
foo(bar="bar", baz="baz", bam="bam", *[])
```

Don't know if testing with a variable name adds an edge case here but if it does can we add :D ...

Reply:
The logic we actually have doesn't depend on using constants vs variable names here, but I'm generally in favor of more-comprehensive testing as a guard against regressions 👍

Reply:
I've kept it.